packages/daemon/src/db/index.ts

```diff
@@ -438,25 +440,20 @@ export const voidTransaction = async (
       // for locked authorities, it doesn't make sense to perform the same operation. The authority needs to be
       // unlocked before it can be spent. In case we're just adding new locked authorities, this will be taken
       // care by the first sql query.
-
-      // update address_tx_history with one entry for each pair (address, token)
-      entries.push(txId);
```

Comment:
This doesn't even makes sense, because this method is called one tx at a time...

Probably lazy code by me when I migrated it from the old daemon